hudi-client/src/main/java/org/apache/hudi/io/HoodieCleanHelper.java

```diff
@@ -176,10 +176,10 @@ public HoodieCleanHelper(HoodieTable<T> hoodieTable, HoodieWriteConfig config) {
    * - Leaves the latest version of the file untouched - For older versions, - It leaves all the commits untouched which
    * has occured in last <code>config.getCleanerCommitsRetained()</code> commits - It leaves ONE commit before this
    * window. We assume that the max(query execution time) == commit_batch_time * config.getCleanerCommitsRetained().
-   * This is 12 hours by default. This is essential to leave the file used by the query thats running for the max time.
+   * This is 5 hours by default. This is essential to leave the file used by the query thats running for the max time.
```

Comment:
can you add explicitly  the assumption (assuming ingestion is running every 30 minutes) here too

Reply:
added